egs2/swbd/asr1/local/data.sh

```diff
@@ -38,7 +38,7 @@ fi
   # SWBD: LDC97S62 LDC2002S09 LDC2002T43 LDC2004T19 LDC2005T19 LDC2004S13 LDC2005S13
 swbd1_dir=${SWBD}/LDC97S62
 eval2000_dir="${SWBD}/LDC2002S09/hub5e_00 ${SWBD}/LDC2002T43"
-fisher_dir="${SWBD}/LDC2004T19 ${SWBD}/LDC2005T19 ${SWBD}/LDC2004S13 ${SWBD}/LDC2005S13"
+fisher_dir="${FISHER}/LDC2004T19 ${FISHER}/LDC2005T19 ${FISHER}/LDC2004S13 ${FISHER}/LDC2005S13"
```

Comment:
Why did you change it from `SWBD` to `FISHER`?
You can keep it as `SWBD` (also in `db.sh`).